vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.h

```diff
@@ -82,6 +82,8 @@ class ReactInstanceWin final : public Mso::ActiveObject<IReactInstanceInternal>
   bool UseDeveloperSupport() const noexcept;
   JSIEngine JsiEngine() const noexcept;
 
+  static void CrashHandler(int fd) noexcept;
```

Comment:
>fd [](http://example.com/codeflow?start=31&length=2)

nit: it would be nice to have a more descriptive parameter name. Is it a file descriptor?